### DISCUSSION TITLE
Caching strapi-plugin-navigation results

### DISCUSSION THREAD

**Discussion by niddu85** [strapi-community/strapi-plugin-rest-cache#29] [Q&A]
Hello,
I'm trying to cache results from strapi-plugin-navigation but I'm not able to properly configure the cache plugin since it needs the content type name to cache (eg api::button.button )
Is there anyone that was able to configure it properly?

**Comment by dagodie** [comment]
Hi,
I am also interested in this topic, any suggestions?
Thank you
